roles/foreman_installer/tasks/packages.yml

```diff
@@ -1,4 +1,13 @@
 ---
+- name: Enable ruby:2.7 module
+  dnf:
+    name: '@ruby:2.7'
+    state: present
+  when:
+    - foreman_repositories_version == "nightly" or 2.5 is version(foreman_repositories_version, '>')
```

Comment:
This tests if 2.5 > f_r_v, which is not what you want ;)
`foreman_repositories_version is version ('2.5', '>=')` is probably more what's wanted here?